### PR TITLE
Add missing variable assignment

### PR DESCRIPTION
**Pull request by rereich27**
Thank you for the quick turnaround.

### CODE REVIEW

**Comment by tillkruss** [comment]
lol version bump and everything. Thanks, great work 👌🏻

**Comment by rereich27** [comment]
Haha, I'm a try hard

File: includes/object-cache.php [hunk 1]
```diff
@@ -595,7 +595,7 @@ public function __construct($fail_gracefully = true)
                 if (defined('WP_REDIS_CLUSTER')) {
                     $this->redis->info(current(array_values(WP_REDIS_CLUSTER)));
```
Comment (tillkruss) [inline includes/object-cache.php:596]: This needs a variable assignment as well.

**Comment by tillkruss** [comment]
Well, congrats on your first issue and pull request 🥳

Typically you don't bump versions on GitHub and leave that to the repo owner, but no worries in this case.

**Comment by rereich27** [comment]
Thank you. Yea, I wasn't sure. Didn't want to look like a newb so I did it.
Really appreciate it. Keep up the good work!

**Comment by tillkruss** [comment]
I'm waiting to hear back from these guys before pushing the new version to wp.org:

https://wordpress.org/support/topic/unable-to-flush-after-plugin-update/#post-12835154
https://wordpress.org/support/topic/cant-clear-cache-after-the-new-update/#post-12835156

**Comment by rereich27** [comment]
Awesome, thank you so much!

**Pull request merged**